spec/lib/task_helpers/exports/reports_spec.rb

```diff
@@ -0,0 +1,67 @@
+describe TaskHelpers::Exports::ServiceDialogs do
```

Comment:
I think this should be `TaskHelpers::Exports::Reports` here

Reply:
Ugh, you're correct. That's what I get for copy/pasting.